src/app/clusters/door-lock-server/door-lock-server-user.cpp

```diff
@@ -684,7 +698,8 @@ bool emberAfDoorLockClusterLockDoorCallback(EndpointId endpoint, app::CommandHan
     return true;
 }
 
-bool emberAfDoorLockClusterUnlockDoorCallback(EndpointId endpoint, app::CommandHandler * commandObj, uint8_t * pin)
+bool emberAfDoorLockClusterUnlockDoorCallback(app::CommandHandler * commandObj, const app::ConcreteCommandPath & commandPath,
+                                              EndpointId endpoint, uint8_t * pin, Commands::UnlockDoor::DecodableType & fields)
```

Comment:
Instead of fields (which seems a bit generic, fields in the action header? fields in the payload?), perhaps refer to this more specifically as `commandData`? This is also what the encoding spec denotes it as.

Reply:
I can do that, sure.